src/modules/BugReporting.ts

```diff
@@ -0,0 +1,212 @@
+import { Platform } from 'react-native';
+import { NativeBugReporting } from '../native';
+import { ArgsRegistry } from '../utils/ArgsRegistry';
+import IBGEventEmitter from '../utils/IBGEventEmitter';
+import InstabugConstants from '../utils/InstabugConstants';
+
+export namespace BugReporting {
+  export import invocationEvent = ArgsRegistry.invocationEvent;
+  export import extendedBugReportMode = ArgsRegistry.extendedBugReportMode;
+  export import reportType = ArgsRegistry.reportType;
+  export import option = ArgsRegistry.option;
+  export import position = ArgsRegistry.position;
+
+  /**
+   * Enables and disables manual invocation and prompt options for bug and feedback.
+   * @param isEnabled
+   */
+  export const setEnabled = (isEnabled: boolean) => {
```

Comment:
I agree, `function` somehow is more readable.

Reply:
The only drawback about `function` is that they cannot be typed, for example, this is a function of type `RequestHandler`:

```js
export const apolloLinkRequestHandler: RequestHandler = (
  operation: Operation,
  forward: NextLink,
) => {
  //...
};
```

To avoid mixing styles, let's stick with arrow functions, what do you think?

Reply:
OK, let's stay consistent then.
Also, in the example you brought up, you don't need to write the parameter types since they are defined by `RequestHandler` already (_this should be applied to all other typed functions_).